app/services/webhook_service.py

```diff
@@ -407,3 +416,17 @@ async def _get_work_platform_category_by_account_id(account_id: str):
             return PlatformCategory.COMMERCE
         else:
             return PlatformCategory.SOCIAL
+
+
+async def update_profile_analytics(webhook_request_data: WebhookRequestData):
+    profile_analytics_event = webhook_request_data.data
+    profile_analytics_event_job_id = profile_analytics_event['job_id']
+
+    if profile_analytics_event_job_id:
```

Comment:
since job id is not optional in pydantic schema, it will always be present
if not present pydantic job creation itself will fail